examples/amazon-bedrock/promptfooconfig.nova-sonic.yaml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://promptfoo.dev/config-schema.json
+description: Amazon Bedrock Nova Sonic with tool use and variables
+
+providers:
+  - id: bedrock:amazon.nova-sonic-v1:0
+    config:
+      inferenceConfiguration:
+        maxTokens: 4096
+        temperature: 0.7
+        topP: 0.95
+      textOutputConfiguration:
+        mediaType: text/plain
+      audioInputConfiguration:
+        mediaType: audio/lpcm
+        sampleRateHertz: 16000
+        sampleSizeBits: 16
+        channelCount: 1
+        encoding: base64
+        audioType: SPEECH
+      audioOutputConfiguration:
+        mediaType: audio/lpcm
+        sampleRateHertz: 24000
+        sampleSizeBits: 16
+        channelCount: 1
+        voiceId: matthew
+        encoding: base64
+        audioType: SPEECH
+      region: us-east-1
+
+prompts:
+  - file://nova_sonic_prompt.js
+
+defaultTest:
+  vars:
+    system_message: You are a helpful assistant. Answer the question based on the audio input.
+  metadata:
+    type: text
+    conversationId: date
+
+tests:
+  - vars:
+      audio_file: file://assets/hello.wav
+    metadata:
+      type: audio
+      conversationId: date
+    assert:
+      - type: llm-rubric
+        value: the date and time is today
+  - vars:
+      audio_file: file://assets/weather.wav
+    metadata:
+      type: audio
+      conversationId: date
+    assert:
+      - type: llm-rubric
+        value: the weather is sunny
```

Comment:
In practice I found this will fail, so I changed it to `contains a weather forecast`.